.agents/skills/check-skill-bridges.ps1

```
@@ -0,0 +1,107 @@
+    $sourceSkillMd = Join-Path $source.FullName "SKILL.md"
+    $codexBridgeSkillMd = Join-Path $codexBridgeRoot "$name/SKILL.md"
+
+    if (-not (Test-Path $codexBridgeSkillMd)) {
+        $errors.Add("Missing Codex bridge SKILL.md for '$name'.")
+    }
+
+    $sourceData = Get-SkillData -SkillMdPath $sourceSkillMd
+    $codexData = Get-SkillData -SkillMdPath $codexBridgeSkillMd
+
+    $expectedSourceSkill = "../../../.agent/skills/$name/SKILL.md"
+    $expectedCodexBridgeRef = "../../../.agents/skills/$name/SKILL.md"
```

PSScriptAnalyzer (1.25.0)

[warning] The variable 'expectedCodexBridgeRef' is assigned but never used.

(PSUseDeclaredVarsMoreThanAssignments)